Automattic/wp-calypso
# Site transfers: Display notice on transfer success

Closes https://github.com/Automattic/dotcom-forge/issues/2580

## Proposed Changes

This pr proposes adding a new success notice when the user successfully completes a site transfer

Backend: D112556-code

## Testing Instructions

1. Visit `/sites?site-transfer-confirm=true`
2. Ensure that you see the following:

![success_transfer_notice](https://github.com/Automattic/wp-calypso/assets/497103/d8881ca2-deab-4ded-8a65-18265a0e1a06)






## Pre-merge Checklist
